<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="hostEdit">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="hostLabel">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="logLabel">
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="portEdit">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QTextEdit" name="logEdit"/>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="2" colspan="2">
     <widget class="QTextEdit" name="dataEdit"/>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="algoBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сортировка символов текста по убыванию</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Разворот текста</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сортировка строк текста по возрастанию</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Статистика по используемым символам в тексте</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="dataLabel">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
